contrib/waybar/patches/update-wireplumber-module.patch

```diff
@@ -0,0 +1,373 @@
+diff --git a/src/modules/wireplumber.cpp b/src/modules/wireplumber.cpp
```

Comment:
all three of these patches should be one patch, just https://github.com/Alexays/Waybar/commit/2326727ccbf0456ccfd631e748955f7f67c44a4e.patch